apps/web/pages/team/[slug].tsx

```diff
@@ -263,7 +275,12 @@ export const getServerSideProps = async (context: GetServerSidePropsContext) =>
     context.params?.orgSlug
   );
   const flags = await getFeatureFlagMap(prisma);
-  const team = await getTeamWithMembers({ slug, orgSlug: currentOrgDomain });
+  const team = await getTeamWithMembers({
+    slug,
+    orgSlug: currentOrgDomain,
+    isTeamView: true,
+    isOrgView: isValidOrgDomain && context.resolvedUrl === "/",
+  });
```

Comment:
Adding flexibility to the `getTeamWithMembers` function to know how to optimize the returned object to avoid extra data that is not going to be used impacting page size